adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala

```diff
@@ -182,31 +182,37 @@ class ADAMContextSuite extends ADAMFunSuite {
   sparkTest("can read a gzipped .vcf file") {
     val path = testFile("test.vcf.gz")
     val vcs = sc.loadVcf(path)
-    assert(vcs.rdd.count === 6)
+    assert(vcs.rdd.count === 7)
```

Comment:
Nope... they appear to have been a compressed version of the VCF file from Hadoop-BAM.